app/assets/javascripts/controllers/credentials/credentials_controller.js

```diff
@@ -4,7 +4,7 @@ ManageIQ.angular.app.controller('CredentialsController', ['$scope', '$attrs', fu
   vm.vmScope = function() {
     return $scope.$eval($attrs.vmScope);
   };
-  
+
```

Comment:
Can you get rid of this file altogether from this PR? since it's not really adding anything.